questionpy/_attempt.py

```diff
@@ -229,6 +231,33 @@ def jinja2(self) -> jinja2.Environment:
     def variant(self) -> int:
         return self.attempt_state.variant
 
+    def use_css(self, path_or_uri: str) -> None:
+        url = urlparse(path_or_uri)
+        if url.scheme in {"https", "qpy"} and path_or_uri not in self._css_files:
+            # path_or_uri is already a supported URI.
+            self._css_files.append(path_or_uri)
+            return
+        if url.scheme:
+            msg = f"Unsupported scheme for CSS file URI: '{url.scheme}'"
+            raise ValueError(msg)
+
+        # path_or_uri is not a URI. Interpret it as a path.
+        if path_or_uri.startswith("/"):
+            msg = "Absolute CSS file paths are not supported."
+            raise ValueError(msg)
```

Comment:
Ich halte hier auch einen neuen `RenderError` für sinnvoll.

Reply:
Alle anderen `RenderErrors` doch vom `QuestionUIRenderer` geworfen, das passt doch hier gar nicht?

Reply:
Genau, damit war gemeint, dass wir diesen Fall im `_RenderErrorCollector` abfangen. 'hierfür' hätte in meinem ursprünglichen Kommentar besser gepasst als 'hier'.